INN/umbrella-energynewsnetwork
# Digest page qa changes

## Changes

This pull request makes the following changes:

- Moves "Go" search button to the end of the search form and changes text to "Search"
- Adds outline border to Ninja forms widgets in digest archive sidebar
- Sets all lists in newsletter signup form to be checked by default
  - This affects all newsletter signup forms on the site. Is this going to be ok with the client?
- Increases checkbox size and label font size
- Updates "Manage subscription" button font to match

![Screen Shot 2020-03-11 at 4 29 25 PM](https://user-images.githubusercontent.com/18353636/76461066-8d5af400-63b5-11ea-8d3f-88923f8e5b3b.png)

![Screen Shot 2020-03-11 at 4 30 25 PM](https://user-images.githubusercontent.com/18353636/76461101-a1065a80-63b5-11ea-8a3d-98cd87845b8e.png)

## Why



For #82

## Testing/Questions

Features that this PR affects:

- Digest page search
- Digest page Nina Forms widget
- Sitewide Mailchimp form


Questions that need to be answered before merging:

- [ ] This PR affects all newsletter signup forms on the site. Is this going to be ok with the client?

Steps to test this PR:

1. View the digest page and make sure the new placement of the search button still works
2. View the Mailchimp signup form and make sure all lists are checked by default
3. View all widgets on digest page sidebar and make sure they look ok
